### DISCUSSION TITLE
Balance the Mishima blades

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#7] [Card Balance] [answered]
Dai-Sho Of The Ancient Emperors (MISHIMA) [R]
EQUIPMENT, UNIQUE, BLADE, FACTION
When attached Warrior kills an enemy in COMBAT with the Dai-Sho, add an Ancient Power card to your hand. Equipped warrior may be given KI powers during your turn for no Action cost.

Mishimese Ceremonial Blade (MISHIMA) [R]
EQUIPMENT, BLADE
[F] +4. Warrior is a SLAYER during [F] combat.

**Comment by 21-ric** [comment]
Logical proposal.👍

**Comment by sjberry82** [comment]
**Mishimese Ceremonial Blade** 
Maybe disable Shoot for the Warrior the Blade is given to.

**Dai-Sho Of The Ancient Emperors **
Ancient Powers are something we made up as a placeholder way before KI Powers. We probably should drop "Ancient Power" and start working with existing KI Powers or ones we can add.

"When attached Warrior kills an enemy in COMBAT the Dai-Sho generates a KI Power. Equipped warrior may be given a KI Power during your turn for no Action cost."

**Reply by framerate** [reply]
For what its worth, my original idea for "Ancient Powers" were to be 3 cards that only generated from it. Since its a "relic" it was an attempt to make it truly unique.

HOWEVER

The 3 Powers themselves were placeholders... was supposed to be replaced with 3 (40%/40%/20%) awesome cards that better fit Mishima lore.

**Comment by Kite80** [comment]
What if Dai-sho allows the entire team to be equipped with ki powers with no action cost, like the darkness flows does for dark gifts? Imo this way it is at the same level as the other corp relics

**Comment by TomekPL-1** [comment, marked as answer]
Ancient powers od Dai-Sho is Unique awesome ability and we definitely shoud keep this! I was thinking to shift only 'give Ki powers without spending actions!' ability from mishimsa blade to Dai-Sho.
And maybe give Dai-Sho small plus to F? It is a blade after all

**Reply by 21-ric** [reply]
Evryone agrees about "Mishimese Ceremonial Blade"?

Mishimese Ceremonial Blade (MISHIMA) [R]
EQUIPMENT, BLADE
[F] +4. Warrior is a SLAYER during [F] combat.

For "Dai-Sho Of The Ancient Emperors".

Dai-Sho Of The Ancient Emperors (MISHIMA) [R]
EQUIPMENT, UNIQUE, BLADE, FACTION
When attached Warrior kills an enemy in COMBAT with the Dai-Sho, add an Ancient Power card to your hand. Equipped warrior may be given KI powers during your turn for no Action cost.

Would you like to add a +2 bonus to [F]?

And/or Mishima Warriors may be given KI powers for no action cost. Like the "darkness flows".

**Reply by TomekPL-1** [reply]
I totally agree with this changes. And +2F for Dai-Sho Of The Ancient Emperors sounds fair.

![daishooftheancientemperors](https://user-images.githubusercontent.com/83038538/117355893-6877e900-aeb3-11eb-9067-3c8b982d39f9.jpg)

Old Dai Sho has +6 to F

**Reply by 21-ric** [reply]
In this case, I could play both cards in my next deck. 👍

**Comment by framerate** [comment]
Updating:

Mishimese Ceremonial Blade (MISHIMA) [R]
EQUIPMENT, BLADE
[F] +4. Warrior is a SLAYER during [F] combat.

Dai-Sho Of The Ancient Emperors (MISHIMA) [R]
EQUIPMENT, UNIQUE, BLADE, FACTION
When attached Warrior kills an enemy in COMBAT with the Dai-Sho, add an Ancient Power card to your hand. Equipped warrior may be given KI powers during your turn for no Action cost.
